src/map.js

```diff
@@ -50,9 +50,12 @@ Map.prototype = map.prototype = {
   }
 };
 
-function map(object, f) {
+function map(object, f, g) {
   var map = new Map;
 
+  // Use Default g if unspecified
+  if(g == null) g = function(d) { return d }
```

Comment:
Nice change! A few considerations:

 * This logic creates a new closure for the identity function every time *map* is invoked and not passed an accessor. This could be avoided by moving the identity function definition outside the map function, similar to how it's done in [d3-axis](https://github.com/d3/d3-axis/blob/5d8dc124bada766313afebf57bfb9796334089de/src/axis.js#L43).
 * Minor point, missing semicolon.